htdocs/accountancy/customer/index.php

```diff
@@ -140,6 +140,9 @@
 	$sql.= " AND l.product_type <= 2";
 
 	dol_syslog('htdocs/accountancy/customer/index.php');
+	// MAX_JOIN_SIZE can be very low (ex: 300000) on some limited configurations (ex: https://www.online.net/fr/hosting/online-perso)
+	// This big SELECT command may exceed the MAX_JOIN_SIZE limit => Therefore we use SQL_BIG_SELECTS=1 to disable the MAX_JOIN_SIZE security
+	$db->query("SET SQL_BIG_SELECTS=1");
```

Comment:
You should add here a test 
if ($db->type == 'mysqli') so the code will be done only for mysql